.buildkite/pipelines/pull_request/osquery_cypress.yml

```diff
@@ -9,3 +9,5 @@ steps:
       automatic:
         - exit_status: '*'
           limit: 1
+    artifact_paths:
+      - "target/kibana-osquery/**/*"
```

Comment:
I am not seeing any artifacts attached to [this job](https://buildkite.com/elastic/kibana-pull-request/builds/16260#50db8346-876e-4f15-b0c3-63c12f510a65) - is that expected?

Reply:
yes, right now, artficats (screenshots) will only appear for failed jobs
![image](https://user-images.githubusercontent.com/5188868/148859122-209690e0-29c2-4629-acfd-269475630b51.png)